classes/Http/Controller/TalkController.php

```diff
@@ -304,20 +323,25 @@ public function processCreateAction(Request $req)
                 'user_id' => (int) $user->getId(),
             ];
 
-            /* @var Locator $spot */
-            $spot = $this->service('spot');
+            $talk = $talk_mapper->build($data);
 
-            $talk_mapper = $spot->mapper(\OpenCFP\Domain\Entity\Talk::class);
-            $talk = $talk_mapper->create($data);
+            $talk->relation('tags', new Collection($tag_entity));
+
+            try {
+                $talk_data = $talk_mapper->save($talk, ['relations' => true]);
+            } catch (\Exception $e) {
+                echo $e->getMessage();
+                die();
```

Comment:
Not quiet sure. Interestingly enough anywhere an exception is thrown, they are always just echoed out and died.